EDI/Overview/03_TipsTricks.md

```diff
@@ -0,0 +1,74 @@
+
+- Ask your business partner for the connection details that specify to which address and with which credentials the data are to be transferred. For example, get to know both the GLN number (Global location number) of your business partner and the GLN number of your company.
+
+- Review this specification for the segments your business partner has defined as required and ensure that you have this data available.   
+
+- Note the following: 
+    - Some segments are mandatory (M) only in case if the optional segment (O) above is to be filled. See the example:
+
+      ![Mandatory segments of the EDIFACT standard](../../Assets/Screenshots/EDI/Overview/OptionalSegment.png "[Mandatory segments of the EDIFACT standard]")
+
+    - There are deviations from the standard depending on the business partner:    
+        - Some segments are required (R), even if it is not mandatory in the EDIFACT standard message definition.  These are then usually indicated with **R** (required) in contrast to **M** (mandatory) of the EDIFACT standard. 
+        - The reverse situation is also possible. Sometimes the business partners do not want to have mandatory segments of the EDIFACT message filled.
+        *Actindo* highlights a segment in red, if mandatory parts are not filled according to the EDIFACT standard. However, *Actindo* still sends the message, even if parts defined as mandatory by EDIFACT are missing.  
+
+- *Actindo* enables you to preview each message structure as a string. For example:
+    UNB+UNOC:3+:14+:14+230810:1443+76817314338981'UNH+67850136066306+ORDERS:D:07A:UN:EAN008'BGM+220+202300001'DTM+8::102'UNS+S'UNT+5+67850136066306'UNZ+1+76817314338981'   
+    The specifications of the major vendors often also show parts of this string as examples, so that you can use the preview for checking the output of your current message definition. Check the specification of your business partner for these examples.
+
+    ![Output examples](../../Assets/Screenshots/EDI/Overview/MessageExample.png "[Output example]")
+
+   This way, you can verify from time to time if you are on the right track. 
+
+
+
+## Checklist for managing the EDIFACT export definitions
+
+Use the following checklist to organize your work. Manage the definitions in the following sequence:  
+
+|To do                                   | Done   
+|-----|------- 
+|1. Create the connection to your business partner, see [Manage connections](../Integration/01_ManageConnections.md).|   |     
+|2. Create a basic definition setting for the header, see [Create basic definition settings](../Operation/01_ManageDefinitions.md#create-basic-definition-settings).|   |   
+|3. Prepare the definition of file settings to make this task easier, see [Prepare definition of file settings](../Operation/01_ManageDefinitions.md#prepare-definition-of-file-settings).|   |   
```

Comment:
Braucht man das für die line items Konfiguration? Wenn nicht, würde ich lassen oder später hinzufügen. Sonst ist es verwirrend. Von mir aus, nur eine Reihenfolge, damit es klappt... So was? 
1. Create connection
2. Create header basic definition settings
3. Prozeduren zur Vorbereitung der Line items settings, welche auch immer (prepare line item export, muss man was in header einstellen, damit ich die line items konfigurieren kann? usw.)
4. Create line items definition settings
5. Manage header filter definitions -> get header, specify document name... (oder muss man das vor der Erstellung der line items definition?)
6. Manage line items definitions -> get line items, specify item number...
7. Assign data
8. Map data

Reply:
Da ich im Text darauf eingehe, lass ich es so